pjlib/include/pj/config.h

```diff
@@ -804,6 +804,19 @@
 #  define PJ_HAS_SEMAPHORE	    1
 #endif
 
+/**
+ * Use dispatch semaphores on Darwin.
+ *
+ * Default: 1 on Darwin, 0 otherwise
+ */
+#ifndef PJ_SEMAPHORE_USE_DISPATCH_SEM
+#   if defined(PJ_DARWINOS)
```

Comment:
a minor, for safety and consistency, please update to `#if defined(PJ_DARWINOS) && PJ_DARWINOS!=0`